zigpy/backups.py

```diff
@@ -100,6 +100,13 @@ def from_dict(cls, obj: dict[str, Any]) -> NetworkBackup:
                 obj["node_info"]["version"] = None
                 version = 1
 
+            # Version 2 introduced the `route_table` and `tx_power` fields
```

Comment:
The `tx_power` field doesn't seem to be included in this PR yet:
```suggestion
            # Version 2 introduced the `route_table` field
```
If you merge the other PR shortly after this, feel free to leave it in.